CHANGES.rst

```diff
@@ -392,6 +392,9 @@ Bug fixes
 
 - ``astropy.coordinates``
 
+  - Fix string representation of ``SkyCoord`` objects transformed into
+    the ``AltAz`` frame
```

Comment:
This is an annoying nag, but changelog entries should reference the relevant issue or issues (see examples elsewhere in the file).

You can fix this and then either leave it as is, or try a fixup rebase.